### ISSUE TITLE
cont. from curseforge page. misaligned icons by 1 pixel issue.

### ISSUE THREAD

**Issue by wayser1** [brittyazel/EnhancedRaidFrames#74]
i'm not exactly sure what you want me to put here because i haven't done this before, but this was the picture with misaligned icons. https://i.imgur.com/8kWPRpB.jpeg - after tweaking it for my 5man raid config i got it to line up for now but typically when the size of the raid frame changes another alignment issue appears. this is what it looks like as of tweaking it the latest time - https://i.imgur.com/Y07YIJg.jpg

i can provide more info as you request it

**Comment by brittyazel** [comment]
Can you test the most recent MASTER branch and let me know if this fixes it for you? I made a few changes that might address htis

**Comment by wayser1** [comment]
i wasn't able to get that version to show up in my addon menu, although it says it's enabled. https://i.imgur.com/D51ekoz.jpg

**Comment by wayser1** [comment]
-> https://i.imgur.com/G7uk4Y6.jpg

**Comment by wayser1** [comment]
actually even the most recent version from curseforge doesnt work for me, i'm going to have to dig up an older version

**Comment by wayser1** [comment]
3.3.6 doesn't work for me. 3.3.5 does.

**Comment by wayser1** [comment]
3.3.5 still off by a pixel this time from the top, not the bottom - https://i.imgur.com/W5vFpJ8.jpeg

**Comment by brittyazel** [comment]
3.3.6 doesn't load? Do you have a lua error? You're the first to report such an issue

**Comment by wayser1** [comment]
not sure if there was a lua error, i can check after raid today or tomorrow

**Comment by brittyazel** [comment]
Just to confirm, what are your icon vertical offset values for those two indicator positions? Are they both set to 0?

**Comment by wayser1** [comment]
i don't think i have the power bar offset option. this is my rejuvenation indicator position https://i.imgur.com/3E14guP.jpg - this is my regrowth one https://i.imgur.com/QAk8NGL.jpg - to get them to line up it seems like i have to adjust the vertical offset depending on whichever way the wind blows.

**Comment by brittyazel** [comment]
Is this for WoW Classic? Someone else today reported a classic specific bug for 3.3.6.

**Comment by wayser1** [comment]
yep it's a classic specific issue

**Comment by brittyazel** [comment]
Good to know. Just pointing out it was in the bug reporting template for you to state if it was classic or retail. I wasted about 2 hours of my work day trying to replicate this on Retail.

That said, please test MASTER now. I have fixed (I think) the classic specific crash, so you can now test the latest code changes.

**Comment by wayser1** [comment]
dude i have no idea about github rules u told me to post here and thats where we're at. i made an account to comment here for you lol

**Comment by wayser1** [comment]
ok word i'll try that after raid thank you

**Comment by wayser1** [comment]
sorry i've been really busy, i should be able to test it today/tmo

**Comment by wayser1** [comment]
hey man sorry for the lack of reply. i've run into this issue again unfortunately and once more i can't determine the cause. this is classic client and it's a 24 person group. can you think of things i should try to stop this from happening? https://i.imgur.com/xyf89B3.jpg

**Comment by brittyazel** [comment]
I honestly have no idea what in the code would cause an issue like what I'm seeing in the screenshot. That goes beyond "rounding error" and is many many pixels off. That looks to be as if the left-most icon is just blatantly ignoring your x/y offset values, which shouldn't even be possible. Do you have another addon that's affecting this maybe? I don't play Classic so I cant replicate your issue personally, and you're the only person that I know of that is experiencing anything like this.

**Comment by wayser1** [comment]
i don't use a raid frame addon, this is just default with a black skin. i can try to find out if it's another addon breaking it but i honestly have no idea where to begin. ill try some different stuff today. i just wish i knew what was actually breaking it because it doesnt always happen and it took a couple weeks to break again :\

**Comment by wayser1** [comment]
it looks it is BlizzardRaidFrameFix. that's the addon that makes it so in combat the raid frames do not break when someone joins/leaves group. i'll look for alternate fixes for that issue then. if it breaks again with this addon off i will let you know.

**Comment by wayser1** [comment]
yeah it broke again with blizzard raidframes disabled and to be honest i have no clue what is causing it

**Comment by wayser1** [comment]
do you have ANY idea as to what i could try to overcome this issue? my healing ui borderline relies on your addon :(

**Comment by brittyazel** [comment]
Whichever two positions you are using to get your left and right icons, try two different ones? Perhaps try top-left and top-right. Also, toggle off the option for power bar vertical offset.

Unfortunately with the information provided I can't see a pattern that would trigger your issue, I see nothing in my code that would cause your issue, and it's in a game I don't play so I cannot test solutions or try to isolate the problem.

**Comment by wayser1** [comment]
unfortunately switching the anchor point doesn't work but i suppose i'm just going to have to get used to switching profiles to get rid of the bug. on another note, I have an addon that dynamically changes to a preset RaidProfile based on groupsize, I was wondering if you have any insight on if it was possible to switch the EnhancedRaidFrames addon profile via a script? If so, I can likely work out the rest. this is so I can have larger indicators if the raidframe size is larger, and smaller ones for a 40man etc.
